projects/dac_fmc_ebz/vcu118/README.md

```diff
@@ -1,7 +1,21 @@
-# AD-DAC-FMC-EBZ reference HDL design for VCU118
+# DAC-FMC-EBZ/VCU118 HDL Project
 
-|||
-| ------ | ------ |
-| Wiki  | https://wiki.analog.com/resources/eval/user-guides/ad-dac-fmc-ebz |
-| FMC Location | FMCP HSPC Slot |
-| Configuration file | ../common/config.tcl |
+## Building the project
+
+The parameters configurable through the `make` command, can be found below, as well as in the **common/config.tcl** file; it contains the default configuration.
+
+```
+cd projects/dac_fmc_ebz/vcu118
+make
+```
+
+If other configurations are desired, then the parameters from the HDL project (see below) need to be changed, as well as the Linux/no-OS project configurations.
+
+The overwritable parameters from the environment:
+
+- ADI_DAC_DEVICE: desired device (currently supported: AD9144, AD9154, AD917x); 
+- ADI_DAC_MODE: configuration mode of JESD parameters supported by the device (check **common/config.tcl**);
+- M: TX number of converters per link
+- L: TX number of lanes per link
+- S: TX number of samples per converter per frame
+- NP: TX number of bits per sample, only 16 is supported
```

Comment:
Put also which is the default device and mode for this project (the #default section and mark also in this list which is the default value of the parameters)